src/dev/tslint/run_tslint_cli.ts

```diff
@@ -17,76 +17,31 @@
  * under the License.
  */
 
-import { resolve } from 'path';
-
 import { createToolingLog } from '@kbn/dev-utils';
-import chalk from 'chalk';
-import execa from 'execa';
 import getopts from 'getopts';
-import Listr from 'listr';
-
-import { Project, PROJECTS } from '../typescript';
 
-class LintFailure {
-  constructor(public project: Project, public error: execa.ExecaError) {}
-}
+import { execInProjects, findProjectsByFlag, Project } from '../typescript';
 
 export function runTslintCliOnTsConfigPaths(tsConfigPaths: string[]) {
   runTslintCli(tsConfigPaths.map(tsConfigPath => new Project(tsConfigPath)));
 }
 
-export function runTslintCli(projects = PROJECTS) {
+export function runTslintCli(projects?: Project[]) {
   const log = createToolingLog('info');
   log.pipe(process.stdout);
 
   const opts = getopts(process.argv.slice(2));
+  projects = projects || findProjectsByFlag(opts.project);
 
   if (!opts.format) {
     process.argv.push('--format', 'stylish');
   }
 
-  const list = new Listr(
```

Comment:
All of this logic was moved into the `execInProjects()` helper so it could be shared